.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,fix-ci]
```

```suggestion
    branches: [main]
```
Remove before merging once we decide to go ahead with the approach in this PR.